CLOUDP-302737: migrated docs from docs team to atlas cli repo (#3713)
mongodb/mongodb-atlas-cli: docs/command/atlas-api.txt

Reviewer: you do know this page is not "clickable" from the table of content nav, right?

Reviewer: it's a question for docs, I think we used to have an extension `toc` to generate the right bits for it but I can't find the code now so not sure why it was removed

   :depth: 1
   :class: singlecol

> `experimental <https://www.mongodb.com/docs/atlas/cli/current/command/atlas-api/>`_: experimental: Access all features of the atlas-admin-apiby using the atlas-cli with the syntax: 'atlas api <tag> <operationId>'.

Reviewer 2: ```suggestion
`experimental <https://www.mongodb.com/docs/atlas/cli/current/command/atlas-api/>`_: experimental: Access all features of the Atlas Administration API by using the Atlas CLI with the syntax: 'atlas api <tag> <operationId>'.
```

> This experimental feature streamlines script development by letting you interact directly with any atlas-admin-api endpoint by using the atlas-cli.

Reviewer 2: ```suggestion
This experimental feature streamlines script development by letting you interact directly with any Atlas Administration API endpoint by using the Atlas CLI.
```

For more information on

> - Atlas Admin Api see: https://www.mongodb.com/docs/atlas/reference/api-resources-spec/v2/

Reviewer 2: ```suggestion
- Atlas Administration API see: https://www.mongodb.com/docs/atlas/reference/api-resources-spec/v2/
```

> - Getting started with the Atlas Admin Api: https://www.mongodb.com/docs/atlas/configure-api-access/#std-label-atlas-admin-api-access

Reviewer 2: ```suggestion
- Getting started with the Atlas Administration API: https://www.mongodb.com/docs/atlas/configure-api-access/#std-label-atlas-admin-api-access
```

This experimental feature streamlines script development by letting you interact 
directly with any {+atlas-admin-api+} endpoint by using the {+atlas-cli+}.

> To learn more about working with the {+atlas-admin-api+}, see :ref:`atlas-admin-api-access`.

Reviewer 2: Delete. These lines appear to be duplicates of the ones above.

Syntax
------

.. code-block::
   :caption: Command Syntax

   atlas api <tag> <operationId> [options]

.. important::
   

>    Both ``<tag>`` and ``<operationId>`` must be in :wikipedia:`camelCase <Camel_case>`.

Reviewer 2: ```suggestion
   Both ``<tag>`` and ``<operationId>`` must be in camelCase.
```

Arguments  
---------  

.. list-table::
   :header-rows: 1
   :widths: 30 30 10 30

   * - Name
     - Type
     - Required
     - Description
   * - <tag>
     - string
     - true

>      - The category of {+atlas-admin-api+} operations in :wikipedia:`camelCase <Camel_case>`. 
>        To find and format the tag, check 
>        the `API documentation <https://www.mongodb.com/docs/atlas/reference/api-resources-spec/v2/>`__ 
>        |url| for the endpoint. It appears after ``#tag/``, but you need to change to camelCase. 

Reviewer 2: ```suggestion
     - The category of Atlas Administration API operations in camelCase . 
       To find and format the tag, check 
       the Atlas Administration API documentation URL 
       for the endpoint. It appears after ``#tag/``, but you need to change to camelCase. For the Atlas API 
       documentation, see: https://www.mongodb.com/docs/atlas/reference/api-resources-spec/v2/.
```

       
       For example, in 
       ``https://www.mongodb.com/docs/atlas/reference/api-resources-spec/v2/#tag/Monitoring-and-Logs``, 
       the tag is ``Monitoring-and-Logs``. In camelCase, it's ``monitoringAndLogs``.
   * - <operationId>
     - string
     - true

>      - The identifier of the {+atlas-admin-api+} endpoint in :wikipedia:`camelCase <Camel_case>`. 

Reviewer 2: ```suggestion
     - The identifier of the Atlas Administration API endpoint in camelCase. 
```
